python/pyxbos/pyxbos/drivers/hvac/point_to_value.json

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
Is this file used in the driver? Or just for development? If its not used in the driver, we can remove it

Reply:
No, but I moved all the json files into a subdirectory for orinda-public-library for future reference.